tests/tests-tax.php

```diff
@@ -265,5 +265,18 @@ public function test_download_is_exclusive_of_tax() {
 	public function test_get_payment_tax() {
 		$this->assertEquals( '0.36', edd_get_payment_tax( $this->_payment_id ) );
 	}
+	
```

Comment:
Ok, can you please move it to the area where the rest of the unit tests for testing payment meta (tax) are located?